djangocms_rest_api/serializers/mapping.py

```diff
@@ -1,10 +1,14 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, print_function, unicode_literals
 
-from cmsplugin_filer_image.cms_plugins import FilerImagePlugin
+from django.conf import settings
 
-from djangocms_rest_api.serializers.filerimageplugin_serializer import FilerImagePluginSerializer
 
-serializer_class_mapping = {
```

Comment:
can we do a plugin_serializer_mapping instead?
So we map plugin types to a serializer, this would make things easier to manage and avoid us having to use the custom dict lookup.